students/hyeoni/users/views.py

```diff
@@ -22,11 +22,13 @@ def post(self, request):
 
             if not re.match('^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{8,}$', user_password):
                 return JsonResponse({"MESSAGE" : "PASSWORD_ERROR"}, status=400)
-
+            
+            salt_password = bcrypt.hashpw(user_password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```

Comment:
변수명은 `salted_password` 보다 `hashed_password`가 좀 더 적합해 보입니다~
```suggestion
            hashed_password = bcrypt.hashpw(user_password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```